hammett/conf/global_settings.py

```diff
@@ -10,3 +10,17 @@
 PERMISSIONS = []
 
 TOKEN = ''
+
+REDIS_PERSISTENCE = {
+    'HOST': '127.0.0.1',
+    'PORT': 6379,
+    'DB': 0,
+    'PASSWORD': None,
+    'KEYS': {
+        'BOT': 'bot_data',
+        'CHAT': 'chat_data',
+        'USER': 'user_data',
+        'CALLBACK': 'callback_data',
+        'CONVERSATIONS': 'conversations',
+    },
```

Comment:
The keys are related with the details of the implementation of the persistence. In other words, we shouldn't expose them to users.